KerbalStuff/app.py

```diff
@@ -101,10 +108,32 @@ def handle_500(e: Exception) -> Tuple[str, int]:
 
 
 @app.errorhandler(404)
-def handle_404(e: Exception) -> Tuple[str, int]:
+def handle_404(e: Exception) -> Union[Tuple[str, int], Response, werkzeug.wrappers.Response]:
+    path = request.path
+    if path.startswith('/api/'):
+        return handle_api_exception(e)
     return render_template("not_found.html"), 404
 
 
+def handle_api_exception(e: Exception) -> Union[Response, werkzeug.wrappers.Response]:
```

Comment:
I think we need `@json_output` here. I tried adding some tests, and the client doesn't know that the format is JSON.

https://github.com/HebaruSan/SpaceDock/runs/785560121

I should have a PR-able change for this momentarily...

Reply:
I decided to just push to your branch since we can still make changes before merge. Now the mypy check passes and there are two tests.

Reply:
You're correct, the mimetype was missing.